lemur/plugins/lemur_linuxdst/plugin.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python
+from lemur.plugins.bases import DestinationPlugin
+from lemur.plugins.lemur_linuxdst import remote_host
+# It is required that you setup certificate based authentication for the destiation host
+
+
+class LinuxDstPlugin(DestinationPlugin):
+
+    title = 'Linux Destination Plugin'
+    slug = 'linux-destination'
+    description = 'Allow the distribution of certificates to a Linux host'
+    version = 1
+
+    author = 'Rick Breidenstein '
+    author_url = 'https://github.com/RickB17/'
+
+    options = [
+        {
+            'name': 'dstHost',
+            'type': 'str',
+            'required': True,
+            'helpMessage': 'This is the host you will be sending the certificate to',
+        },
+        {
+            'name': 'dstPort',
+            'type': 'int',
+            'required': True,
+            'helpMessage': 'This is the port SSHD is running on',
+            'default': '22'
+        },
+        {
+            'name': 'dstUser',
+            'type': 'str',
+            'required': True,
+            'helpMessage': 'The user name to use on the remote host. Hopefully not root.',
+            'default': 'root',
+        },
+        {
+            'name': 'dstPriv',
+            'type': 'str',
+            'required': True,
+            'helpMessage': 'The private key to use for auth',
+            'default': '/root/.shh/id_rsa',
+        },
+        {
+            'name': 'dstPrivKey',
+            'type': 'str',
+            'required': False,
+            'helpMessage': 'The password for the destination private key',
+            'default': 'somethingsecret',
+        },
+        {
+            'name': 'dstDir',
+            'type': 'str',
+            'required': True,
+            'helpMessage': 'This is the directory on the host you want to send the certificate to',
+            'default': '/etc/nginx/certs/'
+        },
+        {
+            "available": [
+                "NGINX",
+                "3File"
+            ],
+            "name": "exportType",
+            "required": True,
+            "value": "NGINX",
+            "helpMessage": "Reference the docs for an explaination of each export type",
+            "type": "select"
+        }
+    ]
+    requires_key = False
+
+    def upload(self, name, body, private_key, cert_chain, options, **kwargs):
+
+        export_type = self.get_option('exportType', options)
+        dst_host = self.get_option('dstHost', options)
+        dst_host_port = self.get_option('dstPort', options)
+        dst_user = self.get_option('dstUser', options)
+        dst_priv = self.get_option('dstPriv', options)
+        dst_priv_key = self.get_option('dstPrivKey', options)
```

Comment:
There's no way to do this currently as we haven't had anything sensitive stored in plugin options to date. If you're interested in adding support for this you would 'catch' the value via the destination schema:
https://github.com/Netflix/lemur/blob/master/lemur/destinations/schemas.py#L28

Right now we don't do anything to the values before storing them if we wanted to add encryption support you could add it via a custom marshmallow field that would encrypt and decrypt the values during serialization and deserialization:
http://marshmallow.readthedocs.io/en/latest/custom_fields.html

You could then add an optionally add a value to the options dictionary `encrypt: True` that would only encrypt plugin options that are sensitive.

Thinking through this, it is quite a bit of work. If you decide to pursue it should be an in a separate PR.